src/mscorlib/shared/System/BitConverter.cs

```diff
@@ -298,6 +476,13 @@ public static bool ToBoolean(byte[] value, int startIndex)
             return value[startIndex] != 0;
         }
 
+        public static bool ToBoolean(ReadOnlySpan<byte> value)
+        {
+            if (value.Length < sizeof(byte))
```

Comment:
To match the existing array overload - the value is read as byte and then normalized to true/false.